susemanager-utils/susemanager-sls/salt/channels/debiankeyring.sls

```diff
@@ -0,0 +1,7 @@
+{%- if pillar.get('mgr_metadata_signing_enabled', false) %}
+/usr/share/keyrings/mgr-archive-keyring.gpg:
+  file.managed:
+    - source:
```

Comment:
I like it more to use an ID with the mgr_ prefix and put the file with the `-name: ` parameter.

Reply:
Ok, changed to use `-name:`.